File: Android_Java/wit-example-ble5/app/src/main/java/com/wit/example/ble5/data/WitSensorKey.java
package com.wit.example.ble5.data;

import com.wit.witsdk.sensor.dkey.DoubleKey;
import com.wit.witsdk.sensor.dkey.StringKey;

/**
 * 传感器标准key常量
 *
 * @Author haungyajun
 * @Date 2022/4/26 15:50 （可以根据需要修改）
 */
public class WitSensorKey {

    public static final StringKey ChipTime = new StringKey("ChipTime", "片上时间", "");
    public static final DoubleKey AccX = new DoubleKey("AccX", "加速度X", "g");
    public static final DoubleKey AccY = new DoubleKey("AccY", "加速度Y", "g");
    public static final DoubleKey AccZ = new DoubleKey("AccZ", "加速度Z", "g");
    public static final DoubleKey AsX = new DoubleKey("AsX", "加速度X", "°/s");
    public static final DoubleKey AsY = new DoubleKey("AsY", "加速度Y", "°/s");
    public static final DoubleKey AsZ = new DoubleKey("AsZ", "加速度Z", "°/s");
    public static final DoubleKey AngleX = new DoubleKey("AngleX", "加速度Z", "°");
    public static final DoubleKey AngleY = new DoubleKey("AngleY", "加速度Z", "°");
    public static final DoubleKey AngleZ = new DoubleKey("AngleZ", "加速度Z", "°");
    public static final DoubleKey HX = new DoubleKey("HX", "磁场X", "μT");
    public static final DoubleKey HY = new DoubleKey("HY", "磁场Y", "μT");
    public static final DoubleKey HZ = new DoubleKey("HZ", "磁场Z", "μT");
    public static final DoubleKey T = new DoubleKey("T", "温度", "℃");

    public static final DoubleKey Q0 = new DoubleKey("Q0", "四元数0", "");
    public static final DoubleKey Q1 = new DoubleKey("Q1", "四元数1", "");
    public static final DoubleKey Q2 = new DoubleKey("Q2", "四元数2", "");
    public static final DoubleKey Q3 = new DoubleKey("Q3", "四元数3", "");

    public static final DoubleKey Rssi = new DoubleKey("Rssi", "信号", "");
    public static final DoubleKey ElectricQuantityPercentage = new DoubleKey("ElectricQuantityPercentage", "电量", "%");
    public static final DoubleKey ElectricQuantity = new DoubleKey("ElectricQuantity", "电量(电压)", "V");
    public static final StringKey VersionNumber = new StringKey("VersionNumber", "版本号", "");

}

File: Unity_C#/Android/Assets/Lib/Device/Device/DKey/WitSensorKey.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Wit.SDK.Modular.Sensor.Modular.DataProcessor.Constant
{
    /// <summary>
    /// 倾角传感器标准key常量
    /// </summary>
    public static class WitSensorKey
    {
        // 芯片时间
        public static string ChipTime { get; } = "ChipTime";

        // 加速度X
        public static string AccX { get; } = "AccX";

        // 加速度Y
        public static string AccY { get; } = "AccY";

        // 加速度Z
        public static string AccZ { get; } = "AccZ";

        // 加速度矢量和
        public static string AccM { get; } = "AccM";

        // 角速度
        public static string AsX { get; } = "AsX";

        // 角速度
        public static string AsY { get; } = "AsY";

        // 角速度
        public static string AsZ { get; } = "AsZ";

        // 角速度Z矢量和
        public static string AsM { get; } = "AsM";

        // 角度X
        public static string AngleX { get; } = "AngleX";

        // 角度Y
        public static string AngleY { get; } = "AngleY";

        // 角度Z
        public static string AngleZ { get; } = "AngleZ";

        // 磁场X
        public static string HX { get; } = "HX";

        // 磁场Y
        public static string HY { get; } = "HY";

        // 磁场Z
        public static string HZ { get; } = "HZ";

        // 磁场矢量和
        public static string HM { get; } = "HM";

        // 温度
        public static string T { get; } = "T";

        // 扩展端口 1
        public static string D0 { get; } = "D0";

        // 扩展端口 2
        public static string D1 { get; } = "D1";

        // 扩展端口 3
        public static string D2 { get; } = "D2";

        // 扩展端口 4
        public static string D3 { get; } = "D3";

        // 气压
        public static string P { get; } = "P";

        // 高度
        public static string H { get; } = "H";

        // 经度
        public static string Lon { get; } = "Lon";

        // 经度度表现形式
        public static string LonDeg { get; } = "LonDeg";

        // 纬度
        public static string Lat { get; } = "Lat";

        // 纬度度表现形式
        public static string LatDeg { get; } = "LatDeg";

        /// <summary>
        /// GPS状态
        /// </summary>
        public static string GPSStatus { get; } = "GPSStatus";


        // GPS高度
        public static string GPSHeight { get; } = "GPSHeight";

        // GPS航向
        public static string GPSYaw { get; } = "GPSYaw";

        // GPS地速
        public static string GPSV { get; } = "GPSV";

        // 四元数0
        public static string Q0 { get; } = "Q0";

        // 四元数1
        public static string Q1 { get; } = "Q1";

        // 四元数2
        public static string Q2 { get; } = "Q2";

        // 四元数3
        public static string Q3 { get; } = "Q3";

        // 卫星数量
        public static string SN { get; } = "SN";

        // 位置定位精度
        public static string PDOP { get; } = "PDOP";

        // 水平定位精度
        public static string HDOP { get; } = "HDOP";

        // 垂直定位精度
        public static string VDOP { get; } = "VDOP";

        // 版本号
        public static string VersionNumber { get; } = "VersionNumber";

        // 序列号
        public static string SerialNumber { get; } = "SerialNumber";


    }
}
